labs/dom/collapse/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+   <head>
+      <meta charset="utf-8">
+      <title>Collapse</title>
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="stylesheet" href="https://bootswatch.com/4/litera/bootstrap.min.css" type="text/css">
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/themes/prism-okaidia.min.css">
+      <link rel="stylesheet" href="../../../../assets/css/basic.css"/>
+      <link rel="stylesheet" href="/assets/css/dom.css"/>
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+   </head>
+   <body>
+      <main role="main" id="main">
+         <h1>Collapse</h1>
+         <p class="lead"> Write some JS that makes the extra info section of each information element dissaapear on the click of the <i class="material-icons">expand_less</i> button. Once collapsed the item doesn't need to reexpand just yet. </p>
```

Comment:
disappear*. The second sentence needs adjustment if you want to force them to play wth the listener in the way you're doing it. "Using only a single click event listener" could be a condition at the start.